### PR TITLE
GrazPlan pasture first revision

### PR DESCRIPTION
**Pull request by her123**
working on #7381

### CODE REVIEW

File: Models/Stock/AnimalGroup.cs [hunk 1]
```diff
@@ -16,16 +15,16 @@ namespace Models.GrazPlan
...
         /// <summary>
-        /// AnimalsDynamicGlb differentiates between the "static" version of the
-        /// model used in GrazFeed and the "dynamic" version used elsewhere
+        /// AnimalsDynamicGlb differentiates between the "static" version of the      
+        /// model used in GrazFeed and the "dynamic" version used elsewhere           
         /// </summary>
         private const bool animalsDynamicGlb = true;
-
+       
         /// <summary>
```
[the remark this answers is not in the document]
Comment (her123) [reply Models/Stock/AnimalGroup.cs:23]: Good points. I'll investigate.

**Comment by hol353** [comment]
There is still a modified Models.csproj file in this PR. Need to get remove it.

**Comment by her123** [comment]
> There is still a modified Models.csproj file in this PR. Need to get remove it.

I think it should have modifications that include the new files.

File: Models/Models.csproj [hunk 2]
```diff
@@ -18,6 +18,24 @@
...
+  
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netcoreapp3.1|AnyCPU'">
+    <NoWarn>1701;1702;0414;0169</NoWarn>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net6.0|AnyCPU'">
+    <NoWarn>1701;1702;1591;0065;0649;0006</NoWarn>
+    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
+  </PropertyGroup>
+
```
[the remark this answers is not in the document]
Comment (her123) [reply Models/Models.csproj:35]: If I choose delete on this file within my pull request will it just revert to the base one in master?

**Comment by hol353** [comment]
In the GitHub diff there are a bunch of changes to PropertyGroups in models.csproj, some of them referencing netcoreapp3.1 (which we don't target anymore and should be removed - but not in this PR) and turning warnings off. Can you just revert models.csproj to the tip revision on the master branch so that models.csproj doesn't even appear in this PR?

File: Models/Models.csproj (hunk 2 above)
[the remark this answers is not in the document]
Comment (hol353) [reply Models/Models.csproj:35]: Nope. You need to revert the file to the tip revision of the master branch.

**Comment by hol353** [comment]
@her123 Your latest push to this PR caused Jenkins to run for an hour and a half before I aborted it. Not sure what happened.

**Comment by her123** [comment]
retest this please jenkins

**Comment by her123** [comment]
retest this please jenkins

**Pull request closed without merging**